openjdk/variant-slim.md

```diff
@@ -0,0 +1,3 @@
+## `%%IMAGE%%:slim`
+
+This image installs the `headless` package of `%%IMAGE%%` and so does not contain the common packages contained in the default tag and only contains the minimal packages needed to run `%%IMAGE%%`. Unless you are working in an environment where *only* the `%%IMAGE%%` image will be deployed and you have space constraints, we highly recommend using the default image of this repository.
```

Comment:
This is going to expand to things like the following:

> This image installs the `headless` package of `s390x/openjdk` and so does not contain the common packages ...

So perhaps the following wording instead?

> This variant installs the `-headless` package of OpenJDK and so is missing many UI-related Java packages and does not contain other common packages ...